aiida/common/escaping.py

```diff
@@ -39,9 +39,14 @@ def escape_for_bash(str_to_escape):
         return ''
 
     str_to_escape = str(str_to_escape)
-
-    escaped_quotes = str_to_escape.replace("'", """'"'"'""")
-    return f"'{escaped_quotes}'"
+    if use_double_quotes:
```

Comment:
It would be good to add the following to the docstring:
```
:param str_to_escape: the string to escape.
:param use_double_quotes: boolean, if ``True``, use double quotes instead of single quotes.
:return: the escaped string.
```